_posts/2018-03-10-meetup-march.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Meetup Summary (March, 2018"
+---
+
+The March Monthly meetup was held at Adlife Plaza. Members started
+streaming in from 1400Hrs.
+
+Black Panther is a recent film based on a Marvel Character by the
+same name. It is basically about a country in Africa, that was
+successfully able to avoid colonization and has been hiding its
+technological prowess from the world.
+
+The movie has been lauded by critics and audiences all over. Some
+of the book club members had also watched the film, and there were
+mixed reactions to it.
+
+The people who didn't like the film felt like it focussed a lot
```

Comment:
This paragraph feels weird. Here's how I'd phrase it:
_ There were people who did not like the film. They felt that it focussed too much on an American's view of what a successful Modern African country should look like. This, they claimed, had a queer side effect of making the movie too similar in appearance, action, plot and feel to American sciFi movies, which is a bit unsettling._